src/Compilers/Core/MSBuildTaskTests/TargetTests.cs

```diff
@@ -265,15 +266,15 @@ public void GenerateEditorConfigCoreEvaluatesMetadata()
             Assert.Equal("_GeneratedEditorConfigMetadata", item.ItemType);
 
             var itemType = item.Metadata.SingleOrDefault(m => m.Name == "ItemType");
-            Assert.NotNull(itemType);
+            AssertEx.NotNull(itemType);
```

Comment:
just curious, why these changes?

Reply:
The new `net5.0` APIs brought in more nullable annotations and it made `SingleOrDefault` return a nullable value. The `Assert.NotNull` class doesn't have the `[NotNull]` attribute hence it was causing the line just below this to give a potential null ref warning. I added a helper that has the attribute so that we don't get the warning here. 

Note: I don't believe our upcoming compiler change will really help here because these APIs are still oblivious.